Documentation/Testing/ProjectTesting.rst

```diff
@@ -28,7 +28,7 @@ The Core mono repository is basically a project setup, having local path
 extensions in `typo3/sysexts/*` instead of the more known and lived `packages/*`
 project folder structure.
 
-.. _testing-projects-structure:
+.. _testing-project-structure:
```

Comment:
Maybe it would be good to keep the old link. No need to create redirects or whatever:
```suggestion
.. _testing-projects-structure:
```